### ISSUE TITLE
Clarify Class II prediction algorithms on docs

### ISSUE THREAD

**Issue by jhundal** [griffithlab/pVACtools#234]
Currently, the pVACtools page lists support for four diff MHC Class II predication algorithms - NetMHCIIpan , SMMalign,  NNalign and, MHCnuggets. The first three were added via IEDB, and MHCnuggets was added outside of the IEDB framework.

However, the IEDB webpage lists support for two additional algorithms that I don't see listed on our website- Combinatorial library and Sturniolo. http://tools.iedb.org/mhcii/help/#Method

Are these two not supported by pVACtools?  If so, why?

**Comment by susannasiebert** [comment]
They’re not currently supported but should be trivial to add as long as the IEDB output format is similar to the other algorithms.

**Comment by susannasiebert** [comment]
Combinatorial library is available standalone using the `comblib` method. Sturniolo is not listed in this table of available methods on the API: http://tools.iedb.org/main/tools-api/ (MHC-II binding command line parameters: table). Could Sturniolo be the `teptiope` method? Or is that something else altogether?

**Comment by susannasiebert** [comment]
According to this: http://tools.iedb.org/auto_bench/mhcii/server/6 `teptiope` seems to correspond to Sturniolo.

**Comment by susannasiebert** [comment]
Both Sturniolo and Combinatorial Library output a `score` instead of a `ic50` value. We need to investigate if and how they might correspond to ic50 values. For Sturniolo the scores might even be negative values. This doesn't fit well into our current paradigm with automatic binding cutoff filtering and the way we aggregate the ic50 scores.

**Comment by malachig** [comment]
yeah will definitely have to think about this.  may have to treat these separately.

**Comment by susannasiebert** [comment]
I was hoping that with adding the percentile feature we would be able to start supporting these algorithms but they output ranks, not percentiles. I'm removing this issue from the 2.0.0 milestone again.
